G2P.pm

```diff
@@ -239,6 +240,12 @@ sub new {
   }
   else {
     $file = $params->{file};
+    open IN, "<",  $file;
+    # supporting panelapp by always filtering by gene symbol if PanelApp file 
+    while (<IN>){
+      $params->{filter_by_gene_symbol} = 1 if (/Model_Of_Inheritance/);
```

Comment:
Since you just need to look this at header, you can just break while after first-line check.
```suggestion
      $params->{filter_by_gene_symbol} = 1 if (/Model_Of_Inheritance/);
      last;
```